types/office-js/index.d.ts

```diff
@@ -15286,23 +16600,27 @@ declare namespace Office {
 
 declare namespace OfficeExtension {
     /** An abstract proxy object that represents an object in an Office document. 
-     * You create proxy objects from the context (or from other proxy objects), add commands to a queue to act on the object, and then synchronize the proxy object state with the document by calling `context.sync()`.
+     * You create proxy objects from the context (or from other proxy objects), add commands to a queue to act on the object, and then synchronize the 
+     * proxy object state with the document by calling `context.sync()`.
      */
     class ClientObject {
         /** The request context associated with the object */
         context: ClientRequestContext;
-        /** Returns a boolean value for whether the corresponding object is a null object. You must call `context.sync()` before reading the isNullObject property. */
+        /** Returns a boolean value for whether the corresponding object is a null object. You must call `context.sync()` before reading the 
+         * isNullObject property. */
```

Comment:
Might as well put the `*/` for these last few methods on the next line. That'll match the style in the rest of the d.ts.